local-cli/start_packager.sh

```diff
@@ -0,0 +1,22 @@
+
+set -e
+
+HEADER="${SRCROOT}/Modules/RCTIPAddress.h"
+
+echo "// This file is generated by a build script." > $HEADER
+echo "// Updated on "`date` >> $HEADER
+echo "" >> $HEADER
+
+ip_address="`ifconfig en0 | grep "inet " | cut -d: -f2 | awk '{print $2}'`"
```

Comment:
This depends strongly on your system (only macbooks on wifi, right?). And if you assume that you're on a mac, you could just use `ipconfig getifaddr en0`